specification.md

```diff
@@ -1489,7 +1491,11 @@ a starting [Event state](#Event-state), it is not used to validate its event pay
 The `start` property defines the workflow starting information. For more information see the [start definition](#Start-Definition) section.
 
 The `schemaVersion` property can be used to set the specific Serverless Workflow schema version to use
-to validate this workflow markup. If not provided the latest released schema version is assumed.
+to validate this workflow markup. The default value of this property is `"latest"`, meaning that 
```

Comment:
What happens to workflows which don't specify a version when a new version of the schema is released with new required parameters?

Reply:
from runtimes point of view version of the spec is more likely not that important as usually they will bind to sdk version is in use. But I am starting to think that maybe schemaVersion should be a required attribute to avoid any confusion.

Reply:
yeah +1 on making it required, but i would like to change it to "specVersion" so its easier to understand. 
this follows also what CE is doing with their "specversion" context attribute